dfx.json

```diff
@@ -18,8 +18,8 @@
       "bind": "127.0.0.1:8000",
       "type": "ephemeral"
     },
-    "messaging": {
-      "providers": ["http://10.12.36.13:8080"],
+    "identity": {
+      "providers": ["http://[2a00:fb01:400:42:5000:60ff:fed5:8464]:8080"],
```

Comment:
probably naive from me, but shouldnt this be 
```
https://rdmx6-jaaaa-aaaaa-aaadq-cai.identity.dfinity.network/
```
instead of the dashboard page?

Reply:
I don't know exactly what this points to, but when it's set up like this you can use `dfx` to make calls against the running canister on our testnet.

Reply:
Ok, if that works that looks good to me!